packages/help-center/src/components/utils.tsx

```diff
@@ -131,3 +130,42 @@ export const matchSupportInteractionId = (
 		return foundMatch;
 	}
 };
+
+export const updateConversationStatuses = (
```

Comment:
Seems like we won't need to export this one. We could also inline it in the `filterAndUpdateConversationsWithStatus` function if that makes it more readable.

Reply:
Updated